models/targeting/genome_classes.py

```diff
@@ -212,3 +213,77 @@ def initialize_target_cut_probabilities(self, dt):
+                    self.domains[key_domain].domain_start = indel_location
+            # if domain ends past indel location
+            if domain.domain_end > indel_location:
+                # re-index it
+                self.domains[key_domain].domain_end += indel_size
+                # if this pulls end past indel location
+                if self.domains[key_domain].domain_end < indel_location:
+                    # set it to indel location
+                    self.domains[key_domain].domain_end = indel_location
+                # if start and end are the same (both location)
+                if self.domains[key_domain].domain_start == self.domains[key_domain].domain_end:
+                    # it has been deleted
+                    deleted_domains.append(domain)
+                else:
+                    # otherwise check if it has targets to be re-indexed
+                    for key_target in target_dict[key_domain].keys():
+                        target = target_dict[key_domain][key_target]
+                        if target.current_start > indel_location:
+                            self.domains[key_domain].targets[key_target].current_start += indel_size
+                            # if the target has been damaged or deleted
+                            if self.domains[key_domain].targets[key_target].current_start < indel_location:
+                                # deal with that later
+                                broken_targets.append(self.domains[key_domain].targets[key_target])
+        # remove all deleted domains
+        for domain in deleted_domains:
+            self.remove_domain(domain)
+        # set new genome
+        self.current_genome = new_genome
+        # delete or fix all broken targets
+        for target in broken_targets:
+            domain_label = target.domain.label
+            # if whole target is deleted
+            if target.current_start + 20 < location:
+                self.domains[domain_label].remove_target(target)
+            # else it is just broken
+            else:
+                # find new pam, set new location, set new sequence
+                new_start = self.find_pam(target.current_start)
+                self.domains[domain_label].targets[target.label].current_start = new_start
+                self.domains[domain_label].targets[target.label].sequence = self.current_genome[new_start:new_start+20]
+
+    def large_deletion(self, target1, target2):
+        """Delete section between two open targets
+        """
+        # make sure cut_positions of targets are up to date
+        target1.set_cut_position()
+        target2.set_cut_position()
+        if target1.cut_position > target2.cut_position:
+            location = target2.cut_position
+            del_size = target1.cut_position - target2.cut_position
+        else:
+            location = target1.cut_position
+            del_size = target2.cut_position - target1.cut_position
```

Comment:
these 6 lines could be simplified to:
`location = min(target1.cut_position, target2.cut_position)`
`del_size = abs(target1.cut_position - target2.cut_position)`